aspnetcore/blazor/debug.md

```diff
@@ -42,6 +42,10 @@ Debugging requires either of the following browsers:
 
 Ensure that firewalls or proxies don't block communication with the debug proxy (`NodeJS` process). For more information, see the [Firewall configuration](#firewall-configuration) section.
 
+For Visual Studio Code users:
+
+When using the [C# for Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) **version 1.23.9 or later**, install the latest [Blazor WASM Debugging Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.blazorwasm-companion). To inspect the installed extensions, open **View** > **Extensions** from the menu bar or select the **Extensions** icon in the **Activity** sidebar.
```

Comment:
We might also want to add that users will be alert of this in the UI so they can install via the prompt as well.

Reply:
Will do ... I'll make an adjustment to it on Saturday morning.